packages/core-suggest/core-suggest.js

```diff
@@ -133,9 +254,52 @@ function onMutation (self) {
       }
     }
   }
+
+  // Send messages for screen readers
+  if (mutations) {
+    for (const mutation of mutations) {
+      if (mutation.attributeName === 'hidden') {
+        if (mutation.target === self) {
+          if (mutation.oldValue === '' && !self._empty) {
+            // Notify screen readers when visible and list has content
+            clearTimeout(self._ariaLiveDelay) // Clear existing delay
+            self._ariaLiveDelay = setTimeout(() => notifyResultsVisible(self, items.length), ARIA_LIVE_DELAY)
+            break // Break loop to avoid duplicate messages
+          }
+        } else {
+          if (mutation.oldValue === null && self._empty) {
+            // Notify screen readers when suggestions are completely hidden by filter
+            notifyResultsEmpty(self)
```

Comment:
Do we need to inspect the `mutations`-object at all to figure out the state? Can't we just inspect `.hidden` and `items`?